src/usr/local/www/guiconfig.inc

```diff
@@ -577,6 +577,25 @@ function genhtmltitle($title, $links=true) {
 	return $bc;
 }
 
+function gen_customwidgettitle_div($widgettitle) {
+	$divstr = '<div class="form-group">';
+	$divstr .= '  <label for="customwidgettitle" class="col-sm-4 control-label">' . gettext('Widget title'). '</label>';
+	$divstr .= '  <div class="col-sm-4">';
+	$divstr .= '    <input type="text" name="customwidgettitle" id="customwidgettitle" value="'. $widgettitle . '" class="form-control" />';
```

Comment:
If I do that, then it keeps "amplifying" the special chars on each edit.
e.g. If I put in some title text that makes an attempt to put the widget title in italics:
```
abc <i>zxc नेपाल </i>
```
It correctly escapes the attempted HTML tags, and the title is rendered literally as I typed it, showing the "<" "i" ">" as literal chars - good.
Then when I try to edit the title again, it fills the input area with:
```
abc &lt;i&gt;zxc नेपाल &lt;/i&gt;
```
and if I save without touching anything, I get that literal string as the new widget title.
Then edit/save again "expands/escapes" another round:
```
abc &amp;lt;i&amp;gt;zxc नेपाल &amp;lt;/i&amp;gt;
```
and so on ad-infinitum.